src/ffmpeg.imageio/CMakeLists.txt

```diff
@@ -3,6 +3,34 @@
 # https://github.com/OpenImageIO/oiio
 
 if (FFmpeg_FOUND)
+   if (LINKSTATIC)
+    if (WIN32)
+        set (_static_suffixes .lib .a)
+    else ()
+        set (_static_suffixes .a)
+    endif ()
+
+    set (_static_libraries_found 0)
+    foreach(_ffmpeg_library IN LISTS FFMPEG_LIBRARIES)
+       get_filename_component(_ext ${_ffmpeg_library} LAST_EXT)
+       list (FIND _static_suffixes ${_ext} _index)
+       if (${_index} GREATER -1)
+         MATH(EXPR _static_libraries_found "${static_libraries_found}+1")
+       endif()
+    endforeach()
+
+    if (${_static_libraries_found} GREATER 0)
+     message("${ColorRed}")
+     message(STATUS "Looks like you are linking against some static ffmpeg libraries: ${FFMPEG_LIBRARIES}")
+     message(STATUS "ffmpeg is a LGPL-licensed library")
+     message(STATUS "When linking statically against a LGPL-licensed library your entire project needs to be under LGPL")
+     message("${ColorReset}")
+    endif()
+
+   endif()
```

Comment:
I like the approach. Danny, can you please clean up the indentation -- they don't always align, and some blocks intend 1 space, some 2, some 4? Let's use a consistent 4 like the rest of the cmake files.